### DISCUSSION TITLE
Issue Forms adding Depends On Value support

### DISCUSSION THREAD

**Discussion by wsmelton** [community/community#4597] [Projects and Issues]
Support the ability for a question or section to only be displayed based on the value of a previous section.

Example:

- I asked "Are you running the latest release of software"
- It may be viable in some repos for them to answer yes or no
- If they answer no, then I'd want another question to "not be greyed out" or "become visible" asking "what version are you using" and would be a textarea.

Not sure if this can be done but could see it come in handy where maybe it is shown on the form but once they submit the issue the content section that did not meet the "depends on value" is not actually included (if that makes sense).

**Comment by ShaMan123** [comment]
+1
I would love it to be like git actions [`if`](https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions), where it is possible to access the context object to get task results matching the task id. `id` is already defined by forms, seems appropriate.

**Reply by ossanna16** [reply]
Great suggestion @ShaMan123! I'm glad you found our community 🚀

**Reply by ShaMan123** [reply]
I see I am not alone with the `if` thoughts. I encountered a number of threads discussing it.
